src/forms/FormLines.tsx

```diff
@@ -51,6 +53,14 @@ export function FormLines(props: FormLinesProps) {
             const clone = cloneElement(child, { isFirst: firstFormHeading });
             firstFormHeading = false;
             return clone;
+          } else if (horizontalLayout && typeof child === "object" && (child as ReactElement).props?.label) {
```

Comment:
Thought: Ah! Wow, I'm kinda surprised this works, but yeah I see what you're doing...

I know it's more work, but I was thinking a more robust approach to this change I think would be:

* Rename `PresentationFieldProps.hideLabel` to `labelStyle` with a type of `"inline" | "hidden" | "above" | "left"`
* Remove `TextFieldProps.inlineLabel` (replaced by `labelStyle=inline`)
* Update `TextFieldBase` line ~175 to show the label above-vs-left-vs-hidden based on the `labelStyle`
* Etc. for `Switch`, `SelectField`, etc.

cc @bdow for thoughts/input 

And, again, it will be a larger diff but I think it will let pages that don't necessarily use `FormLines` (which most do not, unfortunately) still use the `labelStyle=top`.

And, being part of `PresentationProps`, callers would ideally be able to set `labelStyle` just once (i.e. via FormLines) and then not have to repeat the `hideLabel` on each of their wrapped form fields